kytos/core/tcp_server.py

```diff
@@ -44,118 +37,112 @@ def __init__(self, server_address, RequestHandlerClass,
         """
         super().__init__(server_address, RequestHandlerClass,
                          bind_and_activate=False)
-        # Registering the register_event 'endpoint' on the server to be
-        #   accessed on the RequestHandler
         self.controller = controller
-        # self.controller_put_raw_event = controller_put_raw_event
 
     def serve_forever(self, poll_interval=0.5):
-        """Handle requests until an explicit shutdown() request."""
+        """Handle requests until an explicit shutdown() is called."""
         try:
             self.server_bind()
             self.server_activate()
             log.info("Kytos listening at %s:%s", self.server_address[0],
                      self.server_address[1])
             super().serve_forever(poll_interval)
         except Exception:
-            log.error("Maybe you have already a Kytos instance.")
-            log.error('Failed to start Kytos daemon.')
+            log.error('Failed to start Kytos TCP Server.')
             self.server_close()
             raise
 
 
-class KytosOpenFlowRequestHandler(BaseRequestHandler):
+class KytosRequestHandler(BaseRequestHandler):
     """The socket/request handler class for our controller.
 
-    It is instantiated once per connection between each switche and the
+    It is instantiated once per connection between each switch and the
     controller.
-    The setup class will dispatch a KytosEvent (SwitchUp?) on the controller,
-    that will be processed by the Controller Core (or a Core App).
-    The finish class will close the connection and dispatch a KytonEvents
-    (SwitchDown?) on the controller. Or this method will be called by the
-    handler of this (SwitchDown) event?
+    The setup method will dispatch a KytosEvent (kytos/core.connection.new) on
+    the controller, that will be processed by a Core App.
+    The finish method will close the connection and dispatch a KytonEvents
+    (kytos/core.connection.closed) on the controller.
     """
 
+    known_ports = {
+        6633: 'openflow'
+    }
+
     def setup(self):
-        """Method used to create a new connection with client.
+        """Method used to setup the new connection.
 
-        This method builds a new connection, makes a new KytosEvent named
-        'kytos/core.connection.new' and put this in a raw buffer.
+        This method builds a new controller Connection, and places a
+        'kytos/core.connection.new' KytosEvent in the app buffer.
         """
         self.ip = self.client_address[0]
         self.port = self.client_address[1]
 
+        log.info("New connection from %s:%s", self.ip, self.port)
+
         self.connection = Connection(self.ip, self.port, self.request)
```

Comment:
adding #noqa on line 82 did not solve it, since tox complains on tcp_server line 1 (probably when importing..)